src/app/models/form.server.model.ts

```diff
@@ -1140,6 +1139,17 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     ).exec()
   }
 
+  FormSchema.statics.archiveForms = async function (
+    formIds: IFormSchema['_id'],
```

Comment:
typing should be
`formIds: IFormSchema['_id'][]``

Reply:
Thanks for the catch! [05cf6b9](https://github.com/opengovsg/FormSG/pull/6639/commits/05cf6b93ac3cf905fa3e9e86f8ef92b8ad4faba0)